src/test/utils/namespace_context.js

```diff
@@ -137,6 +137,12 @@ class NamespaceContext {
         }
     }
 
+    async delete_files_from_noobaa(type, file_names) {
```

Comment:
@paulxuzhang 
I would also consider adding support and tests for version_ids and delete_markers.
This is a general comment not necessarily for this function.

Reply:
version id support is done in different PR. I will see what I can do.

Reply:
Actually per the design, we will reject any request with version Id for namespace bucket enabled with caching. I will not add the relevant tests in this PR. @guptsanj has been working on handling version id. He can add relevant tests on handling version id.

Reply:
@paulxuzhang 
Alright, I got you.
Just making sure that we won't forget to cover that part.
Since delete_markers and version_ids are a significant portion of the multiple_delete flow logic.